src/components/componentContainer.ts

```diff
@@ -75,10 +96,5 @@ module Plottable {
       this.components().slice().forEach((c: Component) => c.detach());
       return this;
     }
-
-    public remove() {
```

Comment:
It's still there, https://github.com/palantir/plottable/pull/1958/files#diff-039fe0a9f2bd34b0a057ca4ed08395ddR31